.github/workflows/condarise.yaml

```diff
@@ -37,6 +37,12 @@ jobs:
       - name: Build package
         run: conda build conda/hail
 
+      - name: Remove old versions
+        run: |
+          anaconda -t ${{ secrets.ANACONDA_TOKEN }} show cpg/hail 2>&1 \
+          | grep "^   +" | head -n-8 | sed 's#   \+ ##' \
```

Comment:
`head -n-8` -- did you mean `-n 8`?

Reply:
No, I meant `head -n-8`, which is "return all but last 8 lines". The latest packages go last.

Reply:
TIL!
(On macOS, this returns `head: illegal line count -- -8`.)

Reply:
Oh, good catch. I'm using gnu head on my laptop.

Wonder how it will go with the CI macos though.

Reply:
Ah, we need to run the clean-up step once anyway, so will do that for ubuntu only.